javascript-algorithms-and-data-structures/learn-basic-string-and-array-methods-by-building-a-music-playlist-app/index.html

```diff
@@ -2,124 +2,181 @@
+                    fill="#F5F6F7"
+                  />
+                </svg>
+              </button>
+              <button class="repeat">
+                <svg
+                  width="13"
+                  height="15"
+                  viewBox="0 0 13 15"
+                  fill="none"
+                  xmlns="http://www.w3.org/2000/svg"
+                >
+                  <path
+                    d="M6.71667 0L2.16667 2.63099L6.71667 5.26197V2.85208C9.72746 2.96612 12.1333 5.44666 12.1333 8.48999C12.1333 11.606 9.6112 14.132 6.5 14.132C3.3888 14.132 0.866667 11.606 0.866667 8.48999C0.866667 8.19482 0.8893 7.90493 0.932922 7.62199H0.0572743C0.0194992 7.90595 0 8.19569 0 8.48999C0 12.0854 2.91015 15 6.5 15C10.0899 15 13 12.0854 13 8.48999C13 4.96723 10.2062 2.098 6.71667 1.98353V0Z"
+                    fill="#F5F6F7"
+                  />
+                </svg>
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="playlist">
+        <div class="playlist__bar">
+          <div>
+            <svg
+              width="130"
+              height="10"
+              viewBox="0 0 130 10"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path d="M130 1H0M130 9H0" stroke="#F1BE32" stroke-width="2" />
+            </svg>
+          </div>
+          <h2 class="playlist-title">Playlist</h2>
+          <div>
+            <svg
+              width="130"
+              height="10"
+              viewBox="0 0 130 10"
+              fill="none"
+              xmlns="http://www.w3.org/2000/svg"
+            >
+              <path d="M130 1H0M130 9H0" stroke="#F1BE32" stroke-width="2" />
+            </svg>
+          </div>
+          <div>
+            <button class="playlist__close">X</button>
           </div>
-
-          <ul class="playlist-container"></ul>
         </div>
+        <div id="playlist__songs"></div>
```

Comment:
```suggestion
        <ul id="playlist__songs" aria-labelled="playlist"></ul>
```

I would suggest we make the playlist an actual list for the benefit of screen reader users. Also give it an accessible name (which will just be the text of the `h2`).